### ISSUE TITLE
Generation fails for more than one target

### ISSUE THREAD

**Issue by themarwhal** [stackb/bazel-stack-vscode-cc#11]
Our current configuration for github.com/magma/magma is like this with 1 binary target.

```
		"bsv.bazel.buildFlags": [
		],
		"bsv.bazel.testFlags": [
		],
		"bsv.bes.enabled": false,
		"bsv.bzl.codesearch.enabled": false,
		"bsv.bzl.invocation.buildEventPublishAllActions": false,
		"bsv.bzl.invocation.enabled": false,
		"bsv.bzl.invocation.invokeWithBuildEventStreaming": false,
		"bsv.bzl.lsp.enableCodelensStarlarkDebug": false,
		"bsv.bzl.lsp.enableCodelensRun": false,
		"bsv.bzl.remoteCache.enabled": false,
		"bsv.bzl.starlarkDebugger.enabled": false,
		// Update this field with any new targets that need compilation database generation
		"bsv.cc.compdb.targets": [
			"//lte/gateway/c/session_manager:sessiond",
		]
```

This works great as expected, but if I add another binary, I end up with an empty compile_commands.json, with not too much error to go off of.

This is the modified config that fails:
```
		"bsv.cc.compdb.targets": [
			"//lte/gateway/c/session_manager:sessiond",
                        "//lte/gateway/c/connection_tracker/src:main",
		]
```

gist of output I see: https://gist.github.com/themarwhal/b27527151f7a6732f91eb1cd3ebe9ad1

Not sure if there could be an error message somewhere else though.

Thanks!

**Comment by djfm** [comment]
I have the exact same issue, glad to see I'm not alone I have been fiddling with my configuration for hours now!

**Comment by wannesvanloock** [comment]
I'm also experiencing the same issue.

**Comment by lucabergamini** [comment]
Same, the only workaround is to do it for each target independently and cat everything together.

@pcj can we expect a fix for this anytime soon?

**Comment by wannesvanloock** [comment]
I started looking into the issue. It appears that the build event json file has a slightly different structure and I managed to get it running with some minor changes to postprocess.py. I'll create a PR soon.

**Comment by pcj** [comment]
Just seeing this, @wannesvanloock thanks for looking into it.

**Comment by pcj** [comment]
Version 1.2.0 published to marketplace with bugfix #13.
